sites/fast-website/src/app/components/side-navigation/side-navigation.styles.ts

```diff
@@ -19,31 +19,29 @@ export const SideNavigationStyles = css`
     ul {
         list-style-type: none;
         padding: 0 var(--design-unit-large);
-        cursor: pointer;
     }
 
     li {
-        height: calc(var(--design-unit) * 12px);
         display: flex;
         flex-direction: column;
         justify-content: center;
     }
 
+    .icon {
+        fill: var(--accent-foreground-cut-rest);
+    }
+
     li:hover .icon {
         fill: var(--accent-fill-hover);
     }
 
     .scroll-indicator {
-        height: var(--design-unit-large);
-        width: calc(var(--design-unit) * 1px);
-        background-color: var(--neutral-foreground-rest);
+        height: calc(var(--design-unit) * 9px);
+        width: calc(var(--focus-outline-width) * 1px);
+        background-color: var(--accent-foreground-cut-rest);
         border-radius: calc(var(--corner-radius) * 1px);
-        margin: 0 var(--design-unit-large);
-    }
-
-    a:hover .scroll-indicator {
-        height: var(--engaged-height);
-        background-color: var(--accent-foreground-hover);
+        margin: calc(var(--design-unit) * 2px) var(--design-unit-large) 0;
+        transition: all 300ms;
```

Comment:
Only transition the properties that are intended to change:

```suggestion
        transition: 300ms ease-in-out;
        transition-property: background-color, height;
```